daemon/plugin/vendorplugin.go

```diff
@@ -3,50 +3,57 @@ package plugin
 import (
 	"context"
 	"net"
-	"runtime"
 
 	"github.com/go-logr/logr"
 	pb "github.com/openshift/dpu-operator/dpu-api/gen"
+	opi "github.com/opiproject/opi-api/network/evpn-gw/v1alpha1/gen/go"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials/insecure"
 	ctrl "sigs.k8s.io/controller-runtime"
 )
 
 const (
-	VendorPluginDir        string = "/var/run/daemon"
+	VendorPluginDir        string = "/var/run/daemon/vendor-plugin"
 	VendorPluginSocketPath string = VendorPluginDir + "/vendor-plugin.sock"
 )
 
 type VendorPlugin interface {
 	Start() (string, int32, error)
 	Stop()
+	CreateBridgePort(bpr *opi.CreateBridgePortRequest) error
+	DeleteBridgePort(bpr *opi.DeleteBridgePortRequest) error
 }
 
 type DummyPlugin struct {
 	log logr.Logger
 }
 
-func NewDummyPlugin(isDpuMode bool) *DummyPlugin {
+func NewDummyPlugin() *DummyPlugin {
 	return &DummyPlugin{
 		log: ctrl.Log.WithName("VSP"),
 	}
 }
 
-func (v *DummyPlugin) Start() (string, string) {
-	if runtime.GOARCH == "amd64" {
-		return "127.0.0.1", "50051"
-	} else {
-		return "127.0.0.1", "50051"
-	}
+func (v *DummyPlugin) Start() (string, int32, error) {
```

Comment:
we can probably remove this now

Reply:
I will move it to tests since that's where it's used and it's specific to testing.